src/sentry/static/sentry/less/organization.less

```diff
@@ -498,6 +498,45 @@ table.project-list {
   }
 }
 
+.organization-stats {
+  height: 150px;
+  margin-bottom: 20px;
+
+  .bar-chart figure a {
+    height: 150px;
+
+    span {
+      &.accepted {
+        background: @gray-lightest;
+      }
+
+      &.rate-limited {
+        background: @red;
+        min-height: 0;
```

Comment:
I wanted to get rid of the min-height on these since without this, the min-height is 2, which leaves red little caps on all the graphs even when values are 0 which looked a little odd since the red here is to highlight that something bad has happened. So it's a bit misleading. Now, in the normal case where there are no events over rate limit, the graph is much cleaner looking.